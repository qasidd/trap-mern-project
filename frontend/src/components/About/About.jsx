'use strict';

const About =()=>{
return(
    <>
        <h1> About </h1>
        <br></br>
        <h2> Redefining the cinema experience! </h2>
        <br></br>
        <p> This is the ABOUT page for QA cinemas. This page will provide information about movies, listings and upcoming releases. </p>
        <h3> Page Under Construction </h3>
</>

);

}

export default About;