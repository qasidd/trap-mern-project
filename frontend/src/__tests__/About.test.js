import About from '../components/About/About';
import '@testing-library/jest-dom';
import { expect, it } from '@jest/globals';
import { create } from 'react-test-renderer';

it(`It should match the snapshot every time it's rendered`, () => {
    const instance = create(<About/>).toJSON();
    expect(instance).toMatchSnapshot();
})