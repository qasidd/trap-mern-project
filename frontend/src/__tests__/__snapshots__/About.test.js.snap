// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should match the snapshot every time it's rendered 1`] = `
Array [
  <h1>
     About 
  </h1>,
  <br />,
  <h2>
     The Team! 
  </h2>,
  <br />,
  <h3>
     Ayesha Raja
  </h3>,
  <p>
    Email: ayesha.raja@qa.com 
  </p>,
  <h3>
     Daniel Ahuchogu 
  </h3>,
  <p>
    Email: daniel.ahuchogu@qa.com 
  </p>,
  <h3>
     Hamza Pervez 
  </h3>,
  <p>
    Email: hamza.pervez@qa.com 
  </p>,
  <h3>
     Siddartha Gurung 
  </h3>,
  <p>
    Email: siddartha.gurung@qa.com 
  </p>,
  <h3>
     Waleed Tahir 
  </h3>,
  <p>
    Email: waleed.tahir@qa.com 
  </p>,
]
`;
